enhancements/workload-partitioning/rationalizing-configuration.md

```diff
@@ -0,0 +1,460 @@
+---
+title: rationalizing-configuration
+authors:
+  - "@dhellmann"
+  - "@mrunalp"
+  - "@browsell"
+  - "@MarSik"
+  - "@fromanirh"
+reviewers:
+  - "@deads2k"
+  - "@sttts"
+  - maintainers of PAO
+approvers:
+  - "@markmc"
+  - "@derekwaynecarr"
+creation-date: 2021-04-26
+status: implementable
+see-also:
+  - "/enhancements/management-workload-partitioning.md"
+---
+
+# Rationalizing Configuration of Workload Partitioning
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The initial iteration of workload partitioning focused on a short path
+to a minimum viable implementation. This enhancement describes the
+loose ends for preparing the feature for GA at a high level, and
+explains the set of other design documents that need to be written
+separately during the next iteration.
+
+## Motivation
```

Comment:
One other motivation is to allow this capability to be leveraged on non-single node configurations

Reply:
Great point, I've updated the motivation and goal list to include that.